dashboard/app/controllers/application_controller.rb

```diff
@@ -31,27 +31,28 @@ def fix_crawlers_with_bad_accept_headers
     end
   end
 
+  # Enable or disable the rack mini-profiler if the 'pp' query string parameter is set.
+  # pp='disabled' will disable it; any other value will enable it.
+  def maybe_enable_profiler
+    pp = params['pp']
+    if pp
+      ENV['RACK_MINI_PROFILER'] = (pp == 'disabled') ? 'off' : 'on'
+    end
+  end
+
+  # Enable the Rails web console if params['dbg'] is set, or disable it
+  # if params['dbg'] is 'off'.
+  def configure_web_console
+    if params[:dbg]
+      cookies[:dbg] = (params[:dbg] != 'off') ? 'on' : nil
+    end
+    @use_web_console = cookies[:dbg]
+  end
```

Comment:
Methods defined in a scope are not restricted to that scope, so style guide says to define them at the upper level.